### ISSUE TITLE
Error occurred when executing LivePortraitProcess:  OpenCV(4.10.0) D:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\imgwarp.cpp:2825: error: (-215:Assertion failed) src.cols > 0 && src.rows > 0 in function 'cv::warpAffine'

### ISSUE THREAD

**Issue by jokero3answer** [kijai/ComfyUI-LivePortraitKJ#5]
[Diagnostics-1720161440.log](https://github.com/user-attachments/files/16105859/Diagnostics-1720161440.log)

**Comment by jokero3answer** [comment]
![1720161564340](https://github.com/kijai/ComfyUI-LivePortrait/assets/130854108/37f2b6fa-a17b-40a3-8d72-77b7c658c0aa)

**Comment by jokero3answer** [comment]
@kijai

**Comment by vikiiing** [comment]
Same problem

**Comment by Kimdow220807** [comment]
Same problem

**Comment by YacratesWyh** [comment]
same problem

**Comment by MauriceBourdon** [comment]
Same problem

**Comment by 3blackbar** [comment]
The error you're encountering with cv2.warpAffine suggests that the source image (img) being passed to the function is empty or incorrectly sized. To address this issue, we need to ensure that the input image and the transformation matrix are valid and properly sized before calling cv2.warpAffine.

Steps to Debug and Fix the Issue
Print Debug Information:
Add print statements to verify the dimensions of the input image and the transformation matrix just before the cv2.warpAffine call. This will help identify if the input image is empty or if there's an issue with the cropping logic.

Check Image Validity:
Ensure that the input image (img) is not None and has valid dimensions (i.e., width and height are greater than zero).

Inspect Cropping Logic:
Verify that the cropping logic does not produce an empty or invalid region.

**Comment by YacratesWyh** [comment]
https://github.com/KwaiVGI/LivePortrait/blob/main/src/utils/resources/mask_template.png
Copy this to the corresponding place, everyone, it's fixed for me

**Comment by 3blackbar** [comment]
this is different repo, also what is this mystery corresponding place ?

**Comment by YacratesWyh** [comment]
> this is different repo, also what is this mystery corresponding place ?

Obviously the src/utils are the exact place here. Just copy this png to your alert in https://github.com/kijai/ComfyUI-LivePortraitKJ/tree/main/liveportrait/utils/resources.

**Comment by 3blackbar** [comment]
OK man that was it, wow... hey the png in this repo is borked and corrupted, it makes the whole code collapse 
D:\comfyui\ComfyUI\custom_nodes\ComfyUI-LivePortrait\liveportrait\utils\resources

**Comment by MauriceBourdon** [comment]
@YacratesWyh  Many Thanks ! It Works !

**Comment by matanby** [comment]
I face the same error, It looks like a Git LFS issue when cloning the repo:
```
git clone https://github.com/kijai/ComfyUI-LivePortraitKJ
Cloning into 'ComfyUI-LivePortraitKJ'...
remote: Enumerating objects: 128, done.
remote: Counting objects: 100% (128/128), done.
remote: Compressing objects: 100% (108/108), done.
remote: Total 128 (delta 36), reused 95 (delta 15), pack-reused 0
Receiving objects: 100% (128/128), 56.53 KiB | 2.17 MiB/s, done.
Resolving deltas: 100% (36/36), done.
Downloading assets/examples/driving/d0.mp4 (3.0 MB)
Error downloading object: assets/examples/driving/d0.mp4 (63f6f99): Smudge error: Error downloading assets/examples/driving/d0.mp4 (63f6f9962e1fdf6e6722172e7a18155204858d5d5ce3b1e0646c150360c33bed): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to ComfyUI-LivePortraitKJ/.git/lfs/logs/20240705T125710.208442011.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: assets/examples/driving/d0.mp4: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

As a result, all the files stored in LFS are not fetched correctly.

> This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

@kijai :-)

**Comment by xllusion-dong** [comment]
> https://github.com/KwaiVGI/LivePortrait/blob/main/src/utils/resources/mask_template.png Copy this to the corresponding place, everyone, it's fixed for me

That works for me

**Comment by PetitPrinceb612** [comment]
> > this is different repo, also what is this mystery corresponding place ?
> 
> Obviously the src/utils are the exact place here. Just copy this png to your alert in https://github.com/kijai/ComfyUI-LivePortraitKJ/tree/main/liveportrait/utils/resources.
It's solved,Thanks.

**Comment by melMass** [comment]
Fixed in #16 you can redownload the repo

**Comment by kijai** [comment]
So to elaborate what happened: The original repo had marked all the media files to use LFS, which I honestly have never used so I was unaware, people cloning this repo filled my quota hard:

![image](https://github.com/kijai/ComfyUI-LivePortraitKJ/assets/40791699/ba27e5af-462e-4617-a9a5-0ebc316a9c03)

Thus after it was full it failed to download the media files, which included the mask_template.png, causing cv2 to error out when receiving empty image.
